src/DonorDashboards/Repositories/Donations.php

```diff
@@ -245,7 +247,8 @@ protected function getReceiptInfo( $payment ) {
 				}
 
 				$label = html_entity_decode( wp_strip_all_tags( $lineItem->label ) );
-				$value = $lineItem->id === 'paymentStatus' ? $this->getFormattedStatus( $payment->status ) : html_entity_decode( wp_strip_all_tags( $lineItem->value ) );
+				$value = str_ireplace( $lineBreaks, "\r\n", $lineItem->value );
```

Comment:
We don't have to replace anything if we fix the source of the problem.

Reply:
removed: https://github.com/impress-org/givewp/compare/959aaa5b55ed...2f9f4a276fad